src/service/mockup/articleServiceMock.ts

```diff
@@ -90,4 +92,10 @@ export class ArticleServiceMock extends MockService implements ArticleService {
       throw new Error(`Article with id: ${id} not found!`);
     }
   }
+
+  bookmarkStatus(currentUserInfo: UserInfo, article: Article): Promise<boolean> {
+    console.log(currentUserInfo);
+    console.log(article);
```

Comment:
Debug output still present

Reply:
mockservice ->  will be delete in a future pr

Reply:
Ok don't know the plans so i guess it's fine